Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyDoubleSpinCtrl.cpp

```diff
@@ -341,6 +341,20 @@ namespace AzToolsFramework
         return newCtrl;
     }
 
+    void doublePropertySpinboxHandler::ResetGUIToDefaults(PropertyDoubleSpinCtrl* GUI)
+    {
+        GUI->setMinimum(-DBL_MAX);
+        GUI->setMaximum(DBL_MAX);
```

Comment:
Why not use `AZStd::numeric_limits<double>::lowest()` and `AZStd::numeric_limits<double>::max()` here as well?

Reply:
Cuz that's what the original code had and I copied/pasted it :). I can change it to AZStd::numeric_limits.